libs/src/services/schemaValidator/index.js

```diff
@@ -0,0 +1,59 @@
+var Ajv = require('ajv')
+
+const TrackSchema = require('./schemas/trackSchema.json')
+const UserSchema = require('./schemas/userSchema.json')
+
+const TRACK_SCHEMA_TYPE = 'TrackSchema'
+const USER_SCHEMA_TYPE = 'UserSchema'
+
+class SchemaValidator {
+  init () {
```

Comment:
how expensive is this?

Reply:
if we use ajv ~55 ms locally, nonblocking. jsonschema dooesnt pre-validate the schema so it happens when you call validate, so no initial time  cost for that